cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -868,14 +868,6 @@ storage:
           --table nat \
           --to-destination ${PRIVATE_EC2_IPV4}:8181
 
-  {{if not (index .Cluster.ConfigItems "upstream_docker")}}
-  - filesystem: root
-    path: /etc/coreos/docker-1.12
-    mode: 0644
-    contents:
-      inline: yes
-  {{end}}
```

Comment:
We should fix the docker version again. I would like to try out docker 17 as 18 isn't officially supported.

Furthermore the new (better) way of defining versions for those bundled binaries is using `torx`: https://coreos.com/blog/toward-docker-17-in-container-linux